src/tcgwars/logic/impl/gen3/PopSeries3.groovy

```diff
@@ -370,14 +437,15 @@ public enum PopSeries3 implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 50
```

Comment:
Text card is mistranscribed, should do 40 damage instead (text also needs to be updated).

```suggestion
            damage 40
```

![image](https://user-images.githubusercontent.com/1235066/84935530-36994080-b0af-11ea-9e87-2418b3584217.png)